client/layout/masterbar/logged-in.jsx

```diff
@@ -238,81 +238,89 @@ class MasterbarLoggedIn extends Component {
 					/>
 				) : null }
 				<Masterbar>
-					{ this.renderMySites() }
-					<Item
-						tipTarget="reader"
-						className="masterbar__reader"
-						url="/read"
-						icon="reader"
-						onClick={ this.clickReader }
-						isActive={ this.isActive( 'reader' ) }
-						tooltip={ translate( 'Read the blogs and topics you follow' ) }
-						preloadSection={ this.preloadReader }
-					>
-						{ translate( 'Reader', { comment: 'Toolbar, must be shorter than ~12 chars' } ) }
-					</Item>
-					{ ( this.props.isSupportSession || config.isEnabled( 'quick-language-switcher' ) ) && (
-						<AsyncLoad require="./quick-language-switcher" placeholder={ null } />
-					) }
-					{ isWordPressActionSearchFeatureEnabled && (
+					<div className="masterbar__section masterbar__section--left">
```

Comment:
The diff of this file looks complicated but all it does it create the three `masterbar__section` divs.

Reply:
yeah, turning off showing whitespace changes makes this much more readable